src/main/scala/io/qbeast/spark/internal/sources/QbeastBaseRelation.scala

```diff
@@ -38,9 +47,51 @@ object QbeastBaseRelation {
    * @return the QbeastBaseRelation
    */
 
-  def forDeltaTable(tableID: QTableID): QbeastBaseRelation = {
-    val log = SparkDeltaMetadataManager.loadDeltaQbeastLog(tableID)
-    QbeastBaseRelation(log.createRelation, log.qbeastSnapshot.loadLatestRevision)
+  /**
+   * Returns a [[QbeastBaseRelation]] that contains all of the data present
+   * in the table. This relation will be continually updated
+   * as files are added or removed from the table. However, new [[QbeastBaseRelation]]
+   * must be requested in order to see changes to the schema.
+   */
+  def createRelation(sqlContext: SQLContext, table: IndexedTable): BaseRelation = {
+
+    val spark = SparkSession.active
+    val tableID = table.tableID
+    val snapshot = QbeastContext.metadataManager.loadSnapshot(tableID)
+    val schema = QbeastContext.metadataManager.loadCurrentSchema(tableID)
+    val revision = snapshot.loadLatestRevision
+    val columnsToIndex = revision.columnTransformers.map(row => row.columnName).mkString(",")
+    val cubeSize = revision.desiredCubeSize
+    val parameters = Map[String, String](
+      "path" -> table.tableID.toString(),
```

Comment:
I'm not sure now if the path is an option that should be added as well, can we verify it somehow?

Reply:
As an option in createRelation function?

Reply:
Yep. Try to delete it from the parameters if you have time, and see if you encounter any problem.

Reply:
I deleted it and there is no error so far